### DISCUSSION TITLE
Very restrictive installation

### DISCUSSION THREAD

**Discussion by Maelep** [UniversalMediaServer/UniversalMediaServer#2746] [Ideas]
I'm developing a Dockerfile but I'm struggling with the installation process because your project seems very "restrictive" concerning installation. That is to say, no matter where the project is installed, it forces you to use the `WORKDIR` and this causes many problems for a "public" installation.

I'm trying to host some servers for my friends and family where they are only able to access to basic commands (start/restart/stop) and their media folder. 

Basically:
- UMS is installed in `/ums`
- The user will be able to put all media files in `/home/user/media`.

So, I have set their `workdir` at `/home/user` but changing state of the server requires to change the `workdir` and it's not possible (in my case).

I would suggest to add a launch parameters to change file localization. Having configuration file located in the user directory is very problematic.

My problem could be resolving by allowing me to move all system folders/files inside `/ums` and be able to starts and read files from this folder:
```txt
/ums/.config
/ums/.cache
/ums/web
...
```

And allow us to change the logfile location too so I could put it inside the user folder.

**Comment by Nadahar** [comment]
You can specify where the configuration files should reside by using the `profile` command line argument. See the Wiki for details: https://github.com/UniversalMediaServer/UniversalMediaServer/wiki/Command-line-arguments
